package.json

```diff
@@ -89,20 +109,33 @@
           "default": "",
           "description": "Path to the NPL sources. Leave empty to use the current workspace root.",
           "scope": "resource",
-          "order": 0
+          "order": 0,
+          "tags": ["NPL"]
         },
         "NPL.testSources": {
           "type": "string",
           "default": "",
           "description": "Path to the NPL test sources. Leave empty to use the current workspace root.",
           "scope": "resource",
-          "order": 10
+          "order": 10,
+          "tags": ["NPL"]
+        },
+        "NPL.migrationDescriptor": {
+          "type": "string",
+          "description": "Path to the NPL migration descriptor. Determines the sources that get deployed to your applications.",
+          "scope": "resource",
+          "additionalProperties": true,
+          "pattern": "^$|.*yaml/migration\\.yml$",
+          "patternErrorMessage": "The migration descriptor must end with yaml/migration.yml",
+          "order": 15,
+          "tags": ["NPL"]
         },
         "NPL.server.version": {
           "type": "string",
           "default": "latest",
           "description": "Version of the language server to use. Use 'latest' for the most recent version, or run the 'NPL: Select Language Server Version' command for a visual picker with auto-download.",
-          "order": 20
+          "order": 20,
+          "tags": ["NPL"]
```

Comment:
I didn't realize this was part of the schema before (the autocomplete doesn't show it), but this achieves what @mikhail-mozhzherin requested in terms of being able to search for just "NPL" to find all the settings.